src/lib/components/Pagination/Pagination.js

```diff
@@ -84,6 +95,7 @@ Pagination.propTypes = {
 Pagination.defaultProps = {
   options: {},
   overridableId: "",
+  renderBasedOnCurrentSize: false,
```

Comment:
@ntarocco Hi Nicola, thanks for your feedback. I knew prop name was not good and I thought about it for a while, but unfortunately, I did not come up with a better one. To be honest, yours is much better. I will rework the code/docs to reflect this. Additionally, we don't use rebasing, so I am not sure, if I am supposed to merge branch to the main inside of the fork and then you merge it that way, or you merge fork's branch to your main. Sorry to hassle you with this, but after we clarify, for sure it would go smoother in the future. Thanks and I'll let you know once I do this.

Reply:
@ntarocco Hello, done, when you can please take a look.